addons/website/static/src/scss/website.scss

```diff
@@ -390,9 +390,17 @@ table.table_desc tr td {
     @extend .o_half_screen_height;
 }
 
-// Allows custom border radius without contents overflowing.
 .card {
+    // Allows custom border radius without contents overflowing.
     overflow: hidden;
+
+    // Allows opacity filters (background-color option)
+    &[class*='bg-black-'],
+    &[class*='bg-white-'] {
+        .card-body {
+            background-color: transparent !important;
+        }
+    }
```

Comment:
Remove this, we'll think about later: @sbuhl maybe control of the header/footer... :'(

Reply:
@Cocographique